app/search/search-processors/search-terms.js

```diff
@@ -1,5 +1,5 @@
 const cleanupSearchTerms = (terms) => {
-  const cleaned = addFullDogIndexIfMissing(terms.replaceAll('  ', ' ').replaceAll('*', ':*').split(' '))
+  const cleaned = addFullDogIndexIfMissing(terms.replace(/\s+/g, ' ').replaceAll('*', ':*').split(' '))
```

Comment:
much nicer 👍

Reply:
That regex was a bug fix for 478146 where it failed if you had triple spaces in between search terms